.gitlab/build_and_test.sh

```diff
@@ -46,8 +47,20 @@ hostname=${hostname%%[0-9]*}
 BUILD_JOBS=${BUILD_JOBS:-"1"}
 
 # load newer python to try the clingo concretizer
-echo "module load python/3.8.2"
-module load python/3.8.2
+# Corona does not have python 3.8.2
+if [[ "${hostname}" == "corona" ]]; then
+    echo "module load python/3.9.12"
+    module load python/3.9.12
+else
+    echo "module load python/3.8.2"
+    module load python/3.8.2
+fi
+
+# Rocm version specific to Corona.
+if [[ "${hostname}" == "corona" ]]; then
+    echo "module load rocm/5.4.1"
+    module load rocm/5.4.1
+fi
```

Comment:
Combine with the block above.

Reply:
As a follow-on item, could use a spack environment with uberenv to avoid having machine specific modules loads?

Reply:
Probably, but not worth doing this until we update to the latest uberenv which has significant changes including the use of environments.